skyportal/utils/tns.py

```diff
@@ -50,7 +55,8 @@
     'ztfr': 111,
     'ztfi': 112,
 }
-INSTRUMENT_TNS_IDS = {v: k for k, v in TNS_INSTRUMENT_IDS.items()}
+
+FILTER_TNS_IDS = {v: k for k, v in TNS_FILTER_IDS.items()}
```

Comment:
Maybe we should rename those: `SNCOSMO_TO_TNSFILTER`, and `TNSFILTER_TO_SNCOSMO`, or something similar to make it a little more clear. Or keep the current names but just add a `_TO_` in between.
Very minor comment however, feel free to ignore.